plugins/com.google.cloud.tools.eclipse.appengine.newproject/META-INF/MANIFEST.MF

```diff
@@ -6,6 +6,7 @@ Bundle-Version: 0.1.0.qualifier
 Bundle-Vendor: Google Inc.
 Bundle-RequiredExecutionEnvironment: JavaSE-1.7
 Bundle-Localization: plugin
+Bundle-ActivationPolicy: lazy
```

Comment:
We needed it while I was working on this PR: for OSGi service lookup we need a bundle context, and until this entry was added the context was null. I've removed it since the code doing the service lookup has been moved to other bundle.